src/Apps/Order2/Routes/Details/Components/Order2DetailsFulfillmentInfo.tsx

```diff
@@ -0,0 +1,99 @@
+import { Box, Spacer, Text } from "@artsy/palette"
+import type { Order2DetailsFulfillmentInfo_order$key } from "__generated__/Order2DetailsFulfillmentInfo_order.graphql"
+import type React from "react"
+import { useFragment } from "react-relay"
+import { graphql } from "relay-runtime"
+
+interface Props {
+  order: Order2DetailsFulfillmentInfo_order$key
+}
+
+const getShippingContent = (fulfillmentDetails): React.ReactNode => {
```

Comment:
Also non-blocking, but helpful over time. Any details that are secondary to the main thing (the component being exported) are helpful to put below the component vs at the top, because these things typically don't change; rather, the component changes. So future devs can jump right to the primary subject vs wading through minor details at the top.